gr-dtv/lib/atsc/atsc_fpll_impl.cc

```diff
@@ -41,32 +41,39 @@ int atsc_fpll_impl::work(int noutput_items,
                          gr_vector_const_void_star& input_items,
                          gr_vector_void_star& output_items)
 {
+    static const float alpha = 0.01;
```

Comment:
Since we're using C++11, why not make these `constexpr` & not `static`, and allow the compiler to directly substitute them like macros? I think that's roughly how this new `const` variant works ...

Reply:
I looked it up, I like it!  Done.  Just pushed.   I like constexpr.  I'll have to keep an eye out for other places we can use it.  If we merge in 3076 we'll get a little boost in the FPLL too.  There's an inline addition for the nco.sincos() in that PR that'll help with FPLL performance here too.

Reply:
Yeah I'm a convert to `constexpr`; it's a great addition to C++! Simple little things like this & what you're doing reduce compiler challenges & make it much easier for the compiler to produce faster code.